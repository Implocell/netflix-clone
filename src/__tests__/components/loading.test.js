import { render } from "@testing-library/react";

import { Loading } from "../../components";

describe("<Loading />", () => {
	test("should render the loading component", () => {
		const { container, getByTestId } = render(
			<Loading src="/iamges/karl.png" data-testid="loading" />
		);
		expect(getByTestId("loading")).toBeTruthy();
		expect(getByTestId("loading-picture")).toBeTruthy();
		expect(container.firstChild).toMatchSnapshot();
	});

	test("should render the loading releasebody", () => {
		const { container } = render(
			<Loading.ReleaseBody  />
		);
		expect(container.firstChild).toMatchSnapshot();
	});
});
