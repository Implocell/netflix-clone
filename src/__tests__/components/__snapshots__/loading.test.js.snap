// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> should render the loading component 1`] = `
<div
  class="loading__Spinner-sc-11yhc25-0 fcPfoZ"
  data-testid="loading"
>
  <img
    class="loading__Picture-sc-11yhc25-1 siMrc"
    data-testid="loading-picture"
    src="/images/users//iamges/karl.png.png"
  />
</div>
`;

exports[`<Loading /> should render the loading releasebody 1`] = `null`;
